<launch>   

  <arg name="device" default="1" />
  <arg name="show" default="true" />
  <arg name="camera_info" default="fisheye_left.yml" />
  
  <node pkg="tuw_uvc" type="wide_angle_node" name="camera"  output="screen">
    <param name="video_device" value="/dev/video$(arg device)"/>
    <param name="show_camera_image" value="$(arg show)"/>  
    <param name="camera_freeze" value="false"/>  
    <param name="convert_image" value="1"/>    
    <param name="camera_info_url" value="file://$(find tuw_uvc)/cfg/$(arg camera_info)" />
    <param name="frame_id" value="camera"/>
    <!-- 
    <param name="width" value="2048"/>
    <param name="height" value="1536"/>
    <param name="width" value="1600"/>
    <param name="height" value="1200"/>
    <param name="width" value="1280"/>
    <param name="height" value="1024"/>
    -->
    <param name="width" value="640"/>
    <param name="height" value="480"/>
    <!--
    <param name="width" value="1600"/>
    <param name="height" value="1200"/>
    -->
  </node>
  
   <!-- undistore image -->
  <group ns="camera" > 
    <node pkg="image_proc" type="image_proc" name="image_proc" />
  </group>
   
</launch>
